transfer.md

```diff
@@ -58,36 +60,38 @@ There are other equivalents for these tools
 | `docker rmi`    | [`podman rmi`](./docs/podman-rmi.1.md)        |
 | `docker run`    | [`podman run`](./docs/podman-run.1.md)        |
 | `docker save`   | [`podman save`](./docs/podman-save.1.md)      |
+| `docker start`  | [`podman start`](./docs/podman-start.1.md)    |
 | `docker stop`   | [`podman stop`](./docs/podman-stop.1.md)      |
 | `docker tag`    | [`podman tag`](./docs/podman-tag.1.md)        |
+| `docker top`    | [`podman top`](./docs/podman-top.1.md)        |
 | `docker unpause`| [`podman unpause`](./docs/podman-unpause.1.md)|
 | `docker version`| [`podman version`](./docs/podman-version.1.md)|
 | `docker wait`   | [`podman wait`](./docs/podman-wait.1.md)   |
 
-*** Use `podman exec` to enter a container and `podman logs` to view the output of pid 1 of a container.
 **** Use mount to take advantage of the entire linux tool chain rather then just cp.  Read [`here`](./docs/podman-cp.1.md) for more information.
 
 ## Missing commands in podman
 
 Those Docker commands currently do not have equivalents in `podman`:
 
- * `docker container`
- * `docker events`
- * `docker image`
- * `docker network`
- * `docker node`
- * `docker plugin`
- * `docker port`
- * `docker rename`
- * `docker restart`
- * `docker search`
- * `docker secret`
- * `docker service`
- * `docker stack`
- * `docker swarm`
- * `docker system`
- * `docker top`
- * `docker volume`
+| Missing command | Description|
+| :--- | :--- |
+| `docker container`||
+| `docker events`   ||
+| `docker image`    ||
+| `docker network`  ||
+| `docker node`     ||
+| `docker plugin`   |podman does not support plugins.  We recommend you use alternative OCI Runtimes or OCI Runtime Hooks to alter behaviour of podman.|
+| `docker port`     ||
+| `docker rename`   | podman does not support rename, you need to podman rm/podman create to rename a container.|
+| `docker restart`  | podman does not support restart.  We recommend that you put your podman containers into a systemd unit file and use it for restarting applications.|
+| `docker search`   ||
+| `docker secret`   ||
+| `docker service`  ||
+| `docker stack`    ||
+| `docker swarm`    | podman does not support swarm.  We support Kubernetes for orchestration using [CRI-O](https://github.com/kubernetes-incubator/cri-o).|
+| `docker system`   ||
+| `docker volume`   | podman does not support volumes.  Volumes should be built on the host operating system and then volume mounted into the containers.|
```

Comment:
Can we add "See podman mount for more information." or something to that effect?

Reply:
Well this is not about podman mount, more about 
```
podman run -v /source:/dest fedora sh
```
In the docker world they defined special Volume plugins, that allow you to execute things like

```
docker run -v PLUGIN:/DEST fedora sh
```
Which will do not support